nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -36,6 +36,14 @@ def test_plot_surf():
     plot_surf(mesh, bg_map=bg, colorbar=True, cbar_vmin=0,
               cbar_vmax=150, cbar_tick_format="%i")
 
+    # Plot with avg_method
+    plot_surf(mesh, surf_map=rng.standard_normal(size=mesh[0].shape[0]), avg_method='mean')
```

Comment:
Maybe you can rely on a `pytest.mark.parametrize` to lighten the test a bit? Here is an example:

```python
@pytest.mark.parametrize("plot_surf_kwargs", [
    {},
    {"bg": np.random.RandomState(42).standard_normal(size=mesh[0].shape[0])},
    {"bg": np.random.RandomState(42).standard_normal(size=mesh[0].shape[0]), "darkness": 0.5},
    {"bg": np.random.RandomState(42).standard_normal(size=mesh[0].shape[0]), "alpha": 0.5},
])
def test_plot_surf(plot_surf_kwargs):
    mesh = generate_surf()
    # Plot mesh only
    plot_surf(mesh, **plot_surf_kwargs)
```

The idea is that `plot_surf_kwargs` will take successively the value provided in the `parametrize` object